certs/utils.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certs
+
+const (
+	clusterCAKey             = "cluster-ca-key"
+	clusterCACertificate     = "cluster-ca-cert"
+	etcdCAKey                = "etcd-ca-key"
+	etcdCACertificate        = "etcd-ca-cert"
+	frontProxyCAKey          = "front-proxy-ca-key"
+	frontProxyCACertificate  = "front-proxy-ca-cert"
+	serviceAccountPublicKey  = "service-account-public-key"
+	serviceAccountPrivateKey = "service-account-private-key"
+)
+
+func CertificatesToMap(c *Certificates) map[string][]byte {
```

Comment:
what's the reason behind making this a function instead of a method on `*Certficiates`?

Reply:
This was my suggestion, only because I was annoyed by the fact that one direction would be a method and the other direction wouldn't be (without a new type) lol. 

I think this works just as well as `*Certificates.ToMap()`.

Reply:
sgtm

Reply:
I was already in the middle of changing this... might as well adhere to conventions.  Wouldn't want to stick out like a sore 👍